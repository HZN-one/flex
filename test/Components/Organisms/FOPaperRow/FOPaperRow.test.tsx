/**
 * @jest-environment jsdom
 */

import * as React from "react";
import "@testing-library/jest-dom";
import { render } from "@testing-library/react";

import { FOPaperRow } from "@Organisms";

describe("FOPaperRow", () => {
  it("should match snapshot", () => {
    const { container } = render(
      <FOPaperRow
        testID="paper-row-example"
        items={[
          {
            title: "Title",
            children: "Description",
          },
        ]}
      />
    );
    expect(container).toMatchSnapshot();
  });
});
