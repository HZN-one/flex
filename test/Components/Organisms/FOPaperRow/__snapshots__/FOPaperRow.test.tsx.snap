// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FOPaperRow should match snapshot 1`] = `
<div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 flex-paper-2 css-1ps6pg7-MuiPaper-root"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-wrap-xs-nowrap css-xgstj9-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-spacing-xs-3 MuiGrid-wrap-xs-nowrap MuiGrid-grid-xs-true css-mo67vd-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
        >
          <div
            class="MuiBox-root css-19idom"
          >
            <span
              class="MuiTypography-root MuiTypography-overline css-1hbyad5-MuiTypography-root"
              data-testid="typography-paper-row-item-title"
            >
              Title
            </span>
          </div>
          <p
            class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
            data-testid="typography-paper-row-item-children"
          >
            Description
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
